boot/boot_images.cmake

```diff
@@ -59,6 +61,53 @@ ${_efisys_file} 1
 set(ISO_MANUFACTURER "ReactOS Foundation")  # For both the publisher and the preparer
 set(ISO_VOLNAME      "ReactOS")             # For both the Volume ID and the Volume set ID
 
+# ISO image options
+set(ISO_COMMON_OPTIONS
+    -iso-level 4 -publisher ${ISO_MANUFACTURER} -preparer ${ISO_MANUFACTURER} -volid ${ISO_VOLNAME} -volset ${ISO_VOLNAME})
+
+set(ISO_LAYOUT_OPTIONS
+    -duplicates-once -no-cache-inodes -graft-points)
+
+## "El Torito" ISO boot options.
+#
+set(ISO_BOOT_OPTIONS)
+set(ISO_BOOT_REGTEST_OPTIONS) # Useful only for BootCDRegTest
+
+# BIOS-based PC boot entry (x86/x64 only)
+if(ARCH STREQUAL "i386" OR ARCH STREQUAL "amd64")
+    set(ISO_BOOT_OPTIONS
+        -eltorito-platform x86 -eltorito-boot loader/isoboot.bin -no-emul-boot -boot-load-size 4)
+    set(ISO_BOOT_REGTEST_OPTIONS
+        -eltorito-platform x86 -eltorito-boot loader/isobtrt.bin -no-emul-boot -boot-load-size 4)
+endif()
+
+# EFI boot entry
+list(APPEND EFI_SUPPORTED_ARCHS "i386" "amd64" "ia64" "arm" "arm64")
```

Comment:
What's the point in this list? We don't support other architectures

Reply:
See the one that defines the EFIP_PLATFORM_ID above in the file.

Reply:
Indeed I may simplify this by always including the EFI boot entry (or by just checking whether EFI_PLATFORM_ID variable is defined).